var/spack/repos/builtin/packages/thrust/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Thrust(Package):
+    """Thrust is a parallel algorithms library
+    which resembles the C++ Standard Template Library (STL)."""
+
+    homepage = "https://thrust.github.io"
+    url      = "https://github.com/thrust/thrust/archive/1.8.2.tar.gz"
+
+    version('1.8.2', 'fc7fc807cba98640c816463b511fb53f')
+
+    def install(self, spec, prefix):
+        install_tree('doc', join_path(prefix, 'doc'))
+        install_tree('examples', join_path(prefix, 'examples'))
+        install_tree('thrust', join_path(prefix, 'include', 'thrust'))
```

Comment:
If you rebase on develop, you now have the ability to use `prefix.doc`, `prefix.examples`, and `prefix.include.thrust`.